cli/openApiSpec.ts

```diff
@@ -0,0 +1,270 @@
+import fs from 'fs'
+import { Command, Option } from "commander";
+import { ValidatorJSON } from 'convex/values'
+import chalk from 'chalk';
+import { AnalyzedFunction, FunctionSpec, getFunctionSpec } from './utils';
+import prettier from "prettier";
+
+export const openApiSpec = new Command("open-api-spec")
+    .summary("Generate an OpenAPI spec from a Convex function definition")
+    .argument("[filePath]", "The file name of the Convex function definition. If this argument is not provided, we will \nretrieve the function spec from your configured convex deployment.\n"
```

Comment:
nit: get rid of the newlines?

also thoughts on `--inputFile` since it's optional? (I feel like positional args are normally required)